### PR TITLE
move 'mc ilm' to 'mc bucket ilm'

### PR DESCRIPTION
**Pull request by harshavardhana**
update relevant docs and use the same
usage message consistently

### CODE REVIEW

File: docs/minio-client-complete-guide.md [hunk 1]
```diff
@@ -1212,11 +1211,11 @@ mc event remove play/andoria arn:minio:sqs:us-east-1:1:your-queue
 
 <a name="ilm"></a>
 ### Command `ilm`
-``ilm`` provides a convenient way to setup various bucket lifecycle configurations.
+``ilm`` - A convenient way manage bucket lifecycle configuration.
```
Comment (poornas) [inline docs/minio-client-complete-guide.md:1214]:
```suggestion
``ilm`` - A convenient way to  bucket lifecycle configuration.
```

File: cmd/main.go [hunk 2]
```diff
@@ -339,7 +339,6 @@ var appCmds = []cli.Command{
 	diffCmd,
 	rmCmd,
 	eventCmd,
-	ilmCmd,
```
Comment (vadmeste) [inline cmd/main.go:342]: If you want backward compatibility, it is enough to keep this varaible with hidden variable set to true in cli.Command{}

Comment (harshavardhana) [reply cmd/main.go:342]: No backward compatibility needed, we are breaking willfully @vadmeste

**Comment by harshavardhana** [comment]
> Also, we really need to release minio-go before releasing mc because it fixes some issues in lifecycle

We don't need a release unless it's urgent we can simply go mod update the master @vadmeste

**Comment by harshavardhana** [comment]
> We don't need a release unless it's urgent we can simply go mod update the master @vadmeste

minio-go is release v7.0.2

**Pull request merged**